core/business.go

```diff
@@ -98,25 +103,55 @@ func (t *TransferRequest) Run() error {
 		Pause(interval), // will pause a given request for a given interval
 		Transfer(),
 	)
+	TFC.RetriveRequest(t)
```

Comment:
RetriveRequest method helps to fetch the data from database according to request id. Majorly this function will help to secure the system. So from frontend side, we will receive actions and request id. Then this function will execute and initialize the request.